.github/workflows/generate-playground.yml

```diff
@@ -1,17 +1,45 @@
-name: Deploy Chinese
+name: Generate Playground Files
 
 # Controls when the action will run. Triggers the workflow on push or pull request
 # events but only for the master branch
 on:
   push:
     branches: [master]
-  pull_request:
-    branches: [master]
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   # This workflow contains a single job called "build"
-  build:
+  generate-english-playground:
+    # The type of runner that the job will run on
+    runs-on: macos-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Checkout to chinese branch & Generate Chinese
+      - name: Generate English Readme
+        run: |
+          ./generate-playground.sh
+
+      # Commit
+      - name: Commit files
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git commit -m "Generate Playground"
+      ## Push changes to master branch
+      - name: Push changes
+        uses: ad-m/github-push-action@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          branch: "master"
+          force: false
+
+ # This workflow contains a single job called "build"
+  generate-chinese-playground-and-branch:
```

Comment:
Then it's ok for a separate branch to automatic maintain, keep it could help Chinese reader for convenience.